clang/test/ParserHLSL/group_shared.hlsl

```diff
@@ -3,8 +3,8 @@ extern groupshared float f;
 extern float groupshared f; // Ok, redeclaration?
 
 
-// NOTE:lambda is not enabled except for hlsl202x.
-// expected-error@+2 {{expected expression}}
+// expected-warning@+3 {{lambdas are a C++11 extension}}
+// expected-error@+2   {{expected body of lambda expression}}
```

Comment:
Should this work? I don't know much about HLSL, so I have no idea whether this is expected behaviour or a bug in my patch.

Reply:
CC @llvm-beanz